packages/medusa/src/api/routes/store/customers/index.ts

```diff
@@ -124,9 +129,103 @@ export type StoreCustomersRes = {
   customer: Omit<Customer, "password_hash">
 }
 
+/**
+ * @schema StoreCustomersResetPasswordRes
```

Comment:
customer model in reset password response will not include expanded-relations from `StoreCustomersRes`. Creating a separate response for reset password endpoint.